pkgs/top-level/all-packages.nix

```diff
@@ -475,6 +475,8 @@ in
 
   fetchFromSavannah = callPackage ../build-support/fetchsavannah {};
 
+  fetchFromSourcehut = callPackage ../build-support/fetchsourcehut { };
```

Comment:
It's a bit unfortunate that upstream uses (at least) `sourcehut`, `Sourcehut`, and `SourceHut`.
Picking `fetchFromSourcehut` should be fine though :)